index.js

```diff
@@ -1,153 +1,157 @@
-"use strict"
+'use strict';
 
 module.exports = {
-    "env": {
-        "browser": true,
-        "node": false,
-        "es6": true
+    env: {
+        browser: true,
+        node: true,
+        es2022: true,
+        amd: true,
```

Comment:
Do we really want to enable all environments for every project? Wouldn't it be better to explicitly set them to true on a project basis when it's needed?

Reply:
@mscharl We don't need to. Which environments would you enable by default?

Reply:
@mscharl I've added `es2022` as it seems to be necessary, otherwise I'll receive this error message:

```
Parsing error: sourceType 'module' is not supported when ecmaVersion < 2015. Consider adding `{ ecmaVersion: 2015 }` to the parser options
```